app/Models/User.php

```diff
@@ -112,4 +124,32 @@ public function hasAnyRole(): bool
     {
         return count($this->roles->pluck('name')->toArray()) > 0;
     }
+
+    public function getTotalRatingsAttribute()
+    {
+        $labels = array(
+            Rating::CREATIVE_LABEL,
+            Rating::FLEXIBLE_LABEL,
+            Rating::FRIENDLY_LABEL,
+            Rating::HELPFUL_LABEL,
+            Rating::PREPARED_LABEL
+        );
+        $total = collect([
+            $labels[0] => 0,
+            $labels[1] => 0,
+            $labels[2] => 0,
+            $labels[3] => 0,
+            $labels[4] => 0,
+        ]);
+        foreach ($this->ratings as $rating) {
+            $tempStr = decbin($rating->categories);
+            while (strlen($tempStr) < 5) {
+                $tempStr = "0".$tempStr;
+            }
```

Comment:
PHP has a built-in function for this
```suggestion
            $tempStr = str_pad(decbin($rating->categories), 5, "0", STR_PAD_LEFT);
```